app/src/main/res/values/strings.xml

```diff
@@ -71,6 +71,8 @@
     <string name="notification_action_shuffle">Shuffle</string>
     <string name="notification_action_buffering">Buffering</string>
     <string name="notification_action_nothing">Nothing</string>
+    <string name="colorize_notification_title">Colorize notifications</string>
+    <string name="colorize_notification_summary">Use Androids feature to set the color of the notification according to the main color of the thumbnail</string>
```

Comment:
```suggestion
    <string name="notification_colorize_title">Colorize notification</string>
    <string name="notification_colorize_summary">Use Android\'s feature to set the color of the notification according to the main color of the thumbnail</string>
```
Also maybe add a note about the fact that some Android versions and UIs never colorize the notification.

Reply:
Oh, maybe this setting should also be hidden on API <21, since on pre-Lollipop we are sure notifications won't be colorized (I can't find the exact Android version where colorized notifications were added, so I'm assuming Lollipop, if you can find more precise information then use that one)